test/EFCore.PG.FunctionalTests/Query/FullTextSearchDbFunctionsNpgsqlTest.cs

```diff
@@ -196,6 +196,36 @@ public void ToTsQuery_With_Config()
 LIMIT 1");
         }
 
+        [Fact]
+        public void WebsearchToTsQuery()
+        {
+            using (var context = CreateContext())
+            {
+                var tsquery = context.Customers.Select(c => EF.Functions.WebSearchToTsQuery("a OR b")).First();
+                Assert.NotNull(tsquery);
+            }
+
+            AssertSql(
+                @"SELECT websearch_to_tsquery('a OR b')
+FROM ""Customers"" AS c
+LIMIT 1");
+        }
+
+        [Fact]
+        public void WebsearchToTsQuery_With_Config()
```

Comment:
Last nit here. You forgot to change the test method name according to the extension method.